src/Formatter.c

```diff
@@ -448,18 +448,20 @@ ZyanStatus ZydisFormatterSetHook(ZydisFormatter* formatter, ZydisFormatterFuncti
 /* ---------------------------------------------------------------------------------------------- */
 
 ZyanStatus ZydisFormatterFormatInstruction(const ZydisFormatter* formatter,
-    const ZydisDecodedInstruction* instruction, char* buffer, ZyanUSize length,
-    ZyanU64 runtime_address)
+    const ZydisDecodedInstruction* instruction, const ZydisDecodedOperand* operands,
+    ZyanU8 operand_count, char* buffer, ZyanUSize length, ZyanU64 runtime_address)
 {
-     return ZydisFormatterFormatInstructionEx(formatter, instruction, buffer, length,
-         runtime_address, ZYAN_NULL);
+     return ZydisFormatterFormatInstructionEx(formatter, instruction, operands, operand_count,
+         buffer, length, runtime_address, ZYAN_NULL);
 }
```

Comment:
Like already discussed in Discord: let's not have the `Ex` variant -- this one argument just isn't worth the complexity.

Reply:
Ah well the discussion was about the decoder "Ex" variant, which is already removed again.

The formatter variant has been around since the beginning. We can think about removing it in a separate change I guess.

Reply:
Ah yeah sorry, my bad, didn't properly read it and just wanted to tick mental point "get rid of Ex variant". Nevermind! That being said, the getting rid of the formatter `Ex` does seem desirable (even if not intentional by this comment :see_no_evil:).